controllers/Tuit2TagController.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file Controller RESTful Web service API for the tuit2tag resource
+ */
+
+import {Request, Response, Express} from "express";
+import Tuit2TagDao from "../daos/Tuit2TagDao";
+import Tuit2TagControllerI from "../interfaces/Tuit2TagControllerI";
+import Tuit2Tag from "../models/tags/Tuit2Tag";
+
+/**
+ * @class Tuit2TagController Implements RESTful Web service API for the
+ * Tuit2Tag resource.Defines the following HTTP endpoints:
+ * <ul>
+ *     <li>POST /api/tuit2tags to record that a new tuit2tag relationship has
+ *     been added to the database
+ *     </li>
+ *     <li>DELETE /api/tuit2tags/:tagID to record that a tuit2tag relationship
+ *     has been deleted
+ *     </li>
+ * </ul>
+ * @property {Tuit2TagDao} tuit2tagDao Singleton DAO implementing tags CRUD operations
+ * @property {Tuit2TagController} Tuit2TagController Singleton controller implementing
+ * RESTful Web service API
+ */
+
+export default class Tuit2TagController implements Tuit2TagControllerI {
+    private static tuit2tagDao: Tuit2TagDao = Tuit2TagDao.getInstance();
+    private static tuit2tagController: Tuit2TagController | null = null;
+    /**
+     * Creates a singleton controller instance
+     * @param {Express} app express instance to declare the RESTful API
+     * @return Tuit2TagController
+     */
+    public static getInstance = (app: Express): Tuit2TagController => {
+        if (Tuit2TagController.tuit2tagController === null) {
+            Tuit2TagController.tuit2tagController = new Tuit2TagController();
+            app.post("api/tuit2tags", Tuit2TagController.tuit2tagController.createTuit2Tag);
+            app.delete("api/tuit2tags/:t2tID", Tuit2TagController.tuit2tagController.deleteTuit2Tag);
```

Comment:
This file doesn't work for me in postman. Suggested edits to line 38:

app.delete("api/tuit2tags/:t2tID", Tuit2TagController.tuit2tagController.deleteTuit2Tag);